Release Notes/Release Notes v2.11.0-beta1.md

```diff
@@ -0,0 +1,42 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 are:
+
+* .NET Standard 2.0 support
+* Support for `$out` to S3
```

Comment:
Not sure if this is a thing to mention.  It's only for Data Lake, so could confuse people if we don't say that explicitly.

Reply:
Seems simplest to just remove this line. Removed! :)